src/server.js

```diff
@@ -202,6 +203,88 @@ app.get(
   }
 );
 
+function averageGeolocation(coords) {
```

Comment:
As a general practice, we want to avoid committing dead code (code that is not used). When code is committed we expect it to become part of the server binary - and there is no benefit in having unused code that is being deployed. In the worst case, it can even become a security or performance risk (although that's unlikely). Since this code is not being used by anything yet, it should not be committed yet.

I would also encourage starting with a simpler approach. The current team goal is to get an MVP up and running - and that can be done as simply as simply picking the first user's location as the search area. Later down the line, you may find that using the center of mass actually doesn't work that well (or perhaps it does), in which case you will have to undo this work. If you have multiple options with no clear evidence of which is best and why, I would advise to generally prefer to choose the simplest route.

Reply:
By deadcode, do you mean the averageGeolocation function? It is actually being used in the /restaurants get API.
Also, with the way the project was divided, we considered this MVP. I decided to work on this sooner than later because some tasks are blocking my others

Reply:
Do you suggest I hold on with this for now?

Reply:
ah, i see, i missed that usage below. i think the datastore piece is in place - is there a reason to use mock data instead of real data?